From 1991ce605aec4d79a03e366e26b211dda59cd006 Mon Sep 17 00:00:00 2001
From: George McCollister <george.mccollister@gmail.com>
Date: Wed, 7 Oct 2015 14:39:00 -0500
Subject: [PATCH] Don't build man pages

Too many external tools are required to build the man pages. We don't
need them anyway.

Signed-off-by: George McCollister <george.mccollister@gmail.com>
---
 Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 06ebf59..e527f34 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,4 @@
 SUBDIRS = src
-dist_man_MANS = evmctl.1
 
 doc_DATA =  examples/ima-genkey-self.sh examples/ima-genkey.sh examples/ima-gen-local-ca.sh
 EXTRA_DIST = autogen.sh $(doc_DATA)
-- 
2.4.5

